tools/count_missing_type_annotations_utils.sh

```diff
@@ -46,7 +46,7 @@ create_missing_python_type_annotations_report() {
     # This does not include tests, tools, database and bin files.
     # :param $1: The name of the output file.
 
-    flake8 lib/rucio --exclude tools,lib/rucio/tests,lib/rucio/db --output-file $1 --select ANN || true
+    flake8 lib/rucio --exclude tools,lib/rucio/tests,lib/rucio/db,lib/rucio/client,lib/rucio/common --output-file $1 --select ANN || true
```

Comment:
I added the `lib/rucio/rse` folder as well. Yeah, let's talk about the `lib/rucio` one.

Reply:
Summary after the talk: Although the `rucio` module is listed as dependency of the client, not everything is used (e.g. `core`). Since not imported code does not get interpreted, it is fine to add type annotations to these unused parts. Our tests would capture all type annotated places, which is used by the client code.